apps/clabs/mock_data.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Dict
```

Comment:
As we talked during the weekly meeting, please create a controller in the location `apps/controllers/clabernetes.py` and then define the class `C9sController` which will abstract the whole behavior of creating the container labs into Kubernetes. For now, since this is a mock data, you should move all content from the "mock_data.py" and "apps/data/clabs_state.json" to that file.